packages/twenty-server/.env.example

```diff
@@ -78,3 +78,4 @@ FRONTEND_URL=http://localhost:3001
 # IS_CONFIG_VARIABLES_IN_DB_ENABLED=false
 # ANALYTICS_ENABLED=
 # CLICKHOUSE_URL=http://default:clickhousePassword@localhost:8123/twenty
+# CALENDAR_BOOKING_PAGE_ID=
```

Comment:
This is not something people self-hosting or developing locally would typically add so I wouldn't have it in .env.example (which is already not exhaustive)